### PR TITLE
[Feature] multipanetary range

### PR DESCRIPTION
**Pull request by Atralupus**
Replace the multiplanetary charts with range charts

### CODE REVIEW

File: charts/multiplanetary/templates/network.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+        patrolRewardService:
+          {{- toYaml (default $.Values.patrolRewardService .patrolRewardService) | nindent 10 }}
+
+        rudolfService:
+          {{- toYaml (default $.Values.rudolfService .rudolfService) | nindent 10 }}
+
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: heimdall
```
Comment (sky1045) [inline charts/multiplanetary/templates/network.yaml:147]: fixed value

**Review by sky1045 — CHANGES_REQUESTED** [review]

File: charts/multiplanetary/templates/network.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            {{- toYaml (default $.Values.snapshot.partitionReset .snapshot.partitionReset) | nindent 12 }}
+          partition:
+            {{- toYaml (default $.Values.snapshot.partition .snapshot.partition) | nindent 12 }}
+          slackChannel: {{ default $.Values.snapshot.slackChannel .snapshot.slackChannel }}
+          image:
+            {{- toYaml (default $.Values.snapshot.image .snapshot.image) | nindent 12 }}
+          cfDistributionId: "{{ default $.Values.snapshot.cfDistributionId .snapshot.cfDistributionId }}"
+          aws_access_key_id: "{{ default $.Values.snapshot.aws_access_key_id .snapshot.aws_access_key_id }}"
+          aws_secret_access_key: "{{ default $.Values.snapshot.aws_secret_access_key .snapshot.aws_secret_access_key }}"
+          path: "{{ $.Values.clusterName }}/heimdall"
```
Comment (sky1045) [inline charts/multiplanetary/templates/network.yaml:52]: should use `snapshot.path` variable

**Review by sky1045 — CHANGES_REQUESTED** [review]

File: charts/multiplanetary/templates/network.yaml (hunk 2 above)
Comment (Atralupus) [reply charts/multiplanetary/templates/network.yaml]: done!

Comment (sky1045) [reply charts/multiplanetary/templates/network.yaml]: @Atralupus did you also write the value `snapshot.path`?? i couldnt find

Comment (Atralupus) [reply charts/multiplanetary/templates/network.yaml]: no, I didn't... can i write the value like this?

```
    snapshot:
      path: 9c-internal-v2/heimdall
```

Comment (sky1045) [reply charts/multiplanetary/templates/network.yaml]: correct

**Review by sky1045 — APPROVED** [review]

**Pull request merged**